### PR TITLE
GCE/Windows: update and add stable1 test jobs

### PR DESCRIPTION
**Pull request by yujuhong**

### CODE REVIEW

File: config/jobs/kubernetes/sig-gcp/sig-gcp-windows.yaml [hunk 1]
```diff
@@ -126,6 +126,39 @@ periodics:
       - --timeout=120m
       image: gcr.io/k8s-testimages/kubekins-e2e:v20190529-49b63a5-master
 
+- name: ci-kubernetes-e2e-windows-gce-k8sstable1
+  decorate: true
+  extra_refs:
+  - org: yujuhong
```
Comment (dashpole) [inline config/jobs/kubernetes/sig-gcp/sig-gcp-windows.yaml]: you have your own org?

**Comment by dashpole** [comment]
/lgtm
/approve
if I can do that here

File: config/jobs/kubernetes/sig-gcp/sig-gcp-windows.yaml (hunk 1 above)
Comment (yujuhong) [reply config/jobs/kubernetes/sig-gcp/sig-gcp-windows.yaml]: everyone has its own org!

File: config/jobs/kubernetes/sig-gcp/sig-gcp-windows.yaml [hunk 2]
```diff
@@ -10,6 +10,8 @@ presets:
     value: "2"
   - name: KUBERNETES_NODE_PLATFORM
     value: "windows"
+  - name: KUBELET_TEST_ARGS
+    value: "--feature-gates=KubeletPodResources=false"
```
Comment (yujuhong) [inline config/jobs/kubernetes/sig-gcp/sig-gcp-windows.yaml]: /cc @mtaufen

**Comment by yujuhong** [comment]
I added a new commit to fix the tests. PTAL again. Thanks!

**Comment by dashpole** [comment]
/lgtm
We should make sure tests pass when the feature gate is enabled... Did you get a link to the failure?  I'll open a bug for it.

**Comment by yujuhong** [comment]
Need approval
/assign @spiffxp

File: config/jobs/kubernetes/sig-gcp/sig-gcp-windows.yaml (hunk 1 above)
Comment (spiffxp) [reply config/jobs/kubernetes/sig-gcp/sig-gcp-windows.yaml]: we should talk about moving this to kubernetes-sigs

File: config/jobs/kubernetes/sig-gcp/sig-gcp-windows.yaml [hunk 3]
```diff
@@ -126,6 +128,39 @@ periodics:
       - --timeout=120m
       image: gcr.io/k8s-testimages/kubekins-e2e:v20190529-49b63a5-master
 
+- name: ci-kubernetes-e2e-windows-gce-k8sstable1
```
Comment (spiffxp) [inline config/jobs/kubernetes/sig-gcp/sig-gcp-windows.yaml]: we may want to consider using a `fork-per-release` annotation here in concert with [config-forker](https://github.com/kubernetes/test-infra/tree/master/experiment/config-forker)

File: config/jobs/kubernetes/sig-gcp/sig-gcp-windows.yaml (hunk 1 above)
Comment (yujuhong) [reply config/jobs/kubernetes/sig-gcp/sig-gcp-windows.yaml]: Agree. This is in our backlog. We could try moving it to https://github.com/kubernetes-sigs/windows-testing/

File: config/jobs/kubernetes/sig-gcp/sig-gcp-windows.yaml (hunk 3 above)
Comment (yujuhong) [reply config/jobs/kubernetes/sig-gcp/sig-gcp-windows.yaml]: Is there an example that I can copy?

**Pull request merged**